.github/workflows/pre-main.yaml

```diff
@@ -233,7 +234,7 @@ jobs:
 
       # Perform smoke tests.
       - name: 'Test: Run test suites'
-        run: TNF_LOG_LEVEL=${TNF_SMOKE_TESTS_LOG_LEVEL} ./run-cnf-suites.sh -l 'common && !affiliated-certification-container-is-certified-digest'
+        run: TNF_LOG_LEVEL=${TNF_SMOKE_TESTS_LOG_LEVEL} ./run-cnf-suites.sh -l ${SMOKE_TESTS_GINKGO_LABELS_FILTER}
```

Comment:
```suggestion
        run: TNF_LOG_LEVEL=${TNF_SMOKE_TESTS_LOG_LEVEL} ./run-cnf-suites.sh -l "${SMOKE_TESTS_GINKGO_LABELS_FILTER}"
```

Reply:
Done in commit https://github.com/test-network-function/cnf-certification-test/pull/1135/commits/3c982fb0355b5889f46c2a8112e3d4c324ec9f84.